qiita_db/metadata_template.py

```diff
@@ -135,14 +135,14 @@ def _as_python_types(metadata_map, headers):
     return values
 
 
-def _prefix_sample_names_with_id(md_template, study):
+def _prefix_sample_names_with_id(md_template, study_id):
```

Comment:
I disagree with this change. Leave is as it was and provide the study. You state that you don't have the study when extending just the id. Make the caller to pass the study...